automation/scripts/generate-keys-and-ledger.sh

```diff
@@ -301,7 +301,7 @@ PROMPT_KEYSETS="${PROMPT_KEYSETS}n
 # Handle passing the above keyset info into interactive 'coda-network genesis' prompts
 while read input
 do echo "$input"
-  sleep 1
+  sleep 2
```

Comment:
fixes key-generation error i'm experiencing locally which looks to be due to some sort of race condition